finance/accounting.md

```diff
@@ -0,0 +1,124 @@
+# Budget projections and accounting
+
+(accounting:projections)=
+## Budget projections
+
+We have a Google Sheet that can be used to create rough budget projections given some input data about our income and expenses.
+
+```{button-link} https://docs.google.com/spreadsheets/d/1zDO_kqnJ1PH3GWOMks5E_1oIpoAJgseWhj3oCohUVZk/edit#gid=929955044
+:color: primary
+
+Budget projections
+```
+
+Our budget projection model begins with a source of truth about our **current and historical costs and revenue**, and then projects into the future by making assumptions about these numbers.
+It allows us to manually specify costs and revenue in the future, if we **know** that they will happen.
+It also lets us specify repeating numbers each month, and a percentage growth rate for monthly recurring revenues.
+
+### Update our budget projections model
+
+As we get historical data from our accounting statements, we can overwrite the **projections** in this model with **data** from what we actually spend and make.
+To do this, use the [accounting statement table summaries](accounting:summary-tables) and update our budget projections numbers with the corresponding numbers in a given category.
+
+Generally speaking, **cells that are automatically computed are in grey**, while those that are hand-edited are not.
+We treat hand-edited cells as the source of truth.
+
+Here are values to update when we get new monthly accounting data from CS&S.
+
+- **Summary tab: Cash on Hand**. Our Cash on Hand is the source of truth for how much money we have each month. In the `Summary` section of the `Summary` tab, manually overwrite the automatically computed amount with the actual amount for the latest month.
+- **Expenses and income tab: Category actuals**. Our expenses tab sums category totals at the top. Overwrite these summary entries with our actual numbers in the accounting table, doing the best you can to map categories between the two.
+
+(accounting:summary-tables)=
+## Accounting summary tables
+
+We have a Sphinx page that summarizes important accounting information for us.
+It is generated from the [CS&S accounting statements](accounting:statements) that we get monthly.
+You can find this website at the following location:
+
+```{button-link} http://2i2c.org/kpis/finances.html#accounting-tables
+:color: primary
+Accounting tables
+```
+
+(accounting:statements)=
+## Accounting statements
+
+We get monthly accounting reports from our fiscal sponsor, {term}`CS&S`.
+These are placed in a shared Google Drive that CS&S controls.
+
+For those with access, you can find the accounting statement folder here:
+
+```{button-link} https://drive.google.com/drive/u/1/folders/1vM_QX1J8GW5z8W5WemxhhVjcCS2kEovN
```

Comment:
I'm seeing the same `/u/1` part of this link that caused issues over in a recent JHub TC PR. Can we change the drive link to the share link of the folder please?

Reply:
good catch